packages/calcite-components/package.json

```diff
@@ -67,10 +67,11 @@
     "util:copy-assets": "npm run util:copy-icons",
     "util:copy-icons": "cpy \"../../node_modules/@esri/calcite-ui-icons/js/*.json\" \"./src/components/icon/assets/\" --flat",
     "util:generate-t9n-docs-json": "tsx support/generateT9nDocsJSON.ts",
+    "util:generate-t9n-manifest": "tsx support/generateT9nManifest.ts",
     "util:generate-supported-browsers-json": "tsx support/generateSupportedBrowsersJSON.ts",
     "util:hydration-styles": "tsx support/hydrationStyles.ts",
     "util:is-working-tree-clean": "[ -z \"$(git status --porcelain=v1)\" ]",
-    "util:prep-build-reqs": "npm run util:copy-assets",
+    "util:prep-build-reqs": "npm run util:copy-assets  && npm run util:generate-t9n-manifest",
```

Comment:
From what I understand, this script is mainly needed for the translation process and isn’t a strict build requirement. It’s fine to include it here for now, but can you create a follow-up refactor issue to move it into our precommit hook (so it only runs if there’s a staged translation bundle)?

Reply:
Good idea.